_blog.yml

```
@@ -932,4 +932,14 @@
   date: June 23, 2022
   tags:
   - guide
-  - nlp
\ No newline at end of file
+  - nlp
+
+- local: eval-on-the-hub-launch
```

```suggestion
- local: eval-on-the-hub
```

drop the "launch", it's cleaner?

---

(shorter URLs are better usually)